apps/user/components/tips/index.styl

```diff
@@ -0,0 +1,25 @@
+.Tips
+  @extends .flex-container
```

Comment:
Extract whatever is important essential about flex container into a mixin or inline it. This leads to very hard to refactor CSS unless you're extending something that's in the same file and closely conceptually related.